### PR TITLE
Release 1.1.0

### PR DESCRIPTION
**Pull request by TyagiLikeMinds**

### CODE REVIEW

File: packages/integration/lib/src/bloc/user_created_comment/user_created_comment_state.dart [hunk 1]
```diff
@@ -0,0 +1,34 @@
+part of 'user_created_comment_bloc.dart';
+
+abstract class LMFeedUserCreatedCommentState extends Equatable {
+  @override
+  List<Object> get props => [];
+}
+
+final class UserCreatedCommentInitialState
```
Comment (divyanshgandhilm) [inline packages/integration/lib/src/bloc/user_created_comment/user_created_comment_state.dart:8]: Missing LM notations in this class

File: packages/integration/lib/src/views/activity/notification_screen.dart [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+          textStyle: TextStyle(
+            fontSize: 18,
+            fontWeight: FontWeight.bold,
+          ),
+        ),
+      ),
+    );
+  }
+
+  void routeNotification(String route) async {
```
Comment (divyanshgandhilm) [inline packages/integration/lib/src/views/activity/notification_screen.dart:257]: Don't we have this function built in `LMNotificationHandler` ?

File: packages/integration/lib/src/views/media/media_preview_screen.dart [hunk 3]
```diff
@@ -60,10 +63,9 @@ class _LMFeedMediaPreviewScreenState extends State<LMFeedMediaPreviewScreen> {
 
   @override
   Widget build(BuildContext context) {
-    final DateFormat formatter = DateFormat('MMMM d, hh:mm');
     final String formatted = formatter.format(post.createdAt);
-    final LMFeedThemeData feedTheme = LMFeedTheme.of(context);
-    return Scaffold(
+    return widgetUtility.scaffold(
```
Comment (divyanshgandhilm) [inline packages/integration/lib/src/views/media/media_preview_screen.dart:67]: No builder classes support in this screen?

File: packages/integration/lib/src/views/post/post_detail_screen.dart [hunk 4]
```diff
@@ -1344,30 +1467,56 @@ class _LMFeedPostDetailScreenState extends State<LMFeedPostDetailScreen> {
     );
   }
 
-  void handlePostPinAction() async {
-    _postDetailScreenHandler!.postData!.isPinned =
-        !_postDetailScreenHandler!.postData!.isPinned;
-    _postDetailScreenHandler!.rebuildPostWidget.value =
-        !_postDetailScreenHandler!.rebuildPostWidget.value;
+  Future<dynamic> handlePostReportAction() {
+    return showModalBottomSheet(
```
Comment (divyanshgandhilm) [inline packages/integration/lib/src/views/post/post_detail_screen.dart:1471]: Post report should also be configurable for bottom sheet or screen.

File: packages/ui/lib/src/widgets/media/image.dart [hunk 5]
```diff
@@ -49,13 +59,17 @@ class _LMImageState extends State<LMFeedImage> {
 
   @override
   Widget build(BuildContext context) {
-    style = widget.style ?? LMFeedTheme.of(context).mediaStyle.imageStyle;
+    style = widget.style ?? LMFeedTheme.instance.theme.mediaStyle.imageStyle;
     return GestureDetector(
       onTap: () => widget.onMediaTap?.call(),
       child: widget.imageUrl != null
-          ? ClipRRect(
```
Comment (divyanshgandhilm) [inline packages/ui/lib/src/widgets/media/image.dart:56]: This was needed to properly round the image, have we tested if image still gets borderRadius applied?

File: packages/ui/lib/src/utils/constants.dart [hunk 6]
```diff
@@ -1,5 +1,5 @@
 const String kRegexLinksAndTags =
-    r'(?:(?:http|https|ftp|www)\:\/\/)?[a-zA-Z0-9\-\.]+\.[a-zA-Z]{1,}(?::[a-zA-Z0-9]*)?\/?[^\s\n]+|[\w\.-]+@[a-zA-Z\d\.-]+\.[a-zA-Z]{2,}|<<([^<>]+)\|route://member/([a-zA-Z-0-9_]+)>>';
+    r'(?:(?:http|https|ftp|www)\:\/\/)?[a-zA-Z0-9\-\.]+\.[a-zA-Z]{1,}(?::[a-zA-Z0-9]*)?\/?[^\s\n]+|[\w\.-]+@[a-zA-Z\d\.-]+\.[a-zA-Z]{2,}|<<([^<>]+)\|route://user_profile/([a-zA-Z-0-9_]+)>>';
 
 const String uiVersion = '1.3.8';
```
Comment (divyanshgandhilm) [inline packages/ui/lib/src/utils/constants.dart:4]: Make sure we are not using this anywhere.

File: packages/integration/lib/src/widgets/user/user_feed.dart [hunk 7]
```diff
@@ -1,124 +1,157 @@
...
-import 'package:likeminds_feed_flutter_core/src/utils/persistence/user_local_preference.dart';
-import 'package:likeminds_feed_flutter_core/src/utils/typedefs.dart';
-import 'package:likeminds_feed_flutter_core/src/views/media/media_preview_screen.dart';
-import 'package:likeminds_feed_flutter_core/src/views/post/widgets/delete_dialog.dart';
 import 'package:media_kit_video/media_kit_video.dart';
-import 'package:overlay_support/overlay_support.dart';
 
-class LMFeedUserFeedWidget extends StatefulWidget {
-  const LMFeedUserFeedWidget({
+class LMFeedUserCreatedPostListView extends StatefulWidget {
```
Comment (divyanshgandhilm) [inline packages/integration/lib/src/widgets/user/user_feed.dart:8]: Remove `views` from the name of all lists, not appropriate for Flutter widgets.

File: packages/integration/lib/src/widgets/user/user_created_comment_listview.dart [hunk 8]
```diff
@@ -0,0 +1,846 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:infinite_scroll_pagination/infinite_scroll_pagination.dart';
+import 'package:likeminds_feed_flutter_core/likeminds_feed_core.dart';
+import 'package:likeminds_feed_flutter_core/src/bloc/user_created_comment/user_created_comment_bloc.dart';
+import 'package:media_kit_video/media_kit_video.dart';
+import 'package:overlay_support/overlay_support.dart';
+
+class LMFeedUserCreatedCommentListView extends StatefulWidget {
```
Comment (divyanshgandhilm) [inline packages/integration/lib/src/widgets/user/user_created_comment_listview.dart:9]: Remove `views` from the name of all lists, not appropriate for Flutter widgets.

File: packages/integration/lib/src/widgets/user/saved_post_listview.dart [hunk 9]
```diff
@@ -0,0 +1,858 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:infinite_scroll_pagination/infinite_scroll_pagination.dart';
+import 'package:likeminds_feed_flutter_core/likeminds_feed_core.dart';
+import 'package:likeminds_feed_flutter_core/src/bloc/saved_post/saved_post_bloc.dart';
+import 'package:media_kit_video/media_kit_video.dart';
+
+class LMFeedSavedPostListView extends StatefulWidget {
```
Comment (divyanshgandhilm) [inline packages/integration/lib/src/widgets/user/saved_post_listview.dart:8]: Remove `views` from the name of all lists, not appropriate for Flutter widgets.

File: packages/integration/lib/src/widgets/lists/feed/feed_list.dart [hunk 10]
```diff
@@ -0,0 +1,672 @@
...
+                    context, defPostWidget(context, feedThemeData, item), item);
+              },
+            ),
+          );
+        },
+      ),
+    );
+  }
+
+  LMFeedPostWidget defPostWidget(BuildContext context,
```
Comment (divyanshgandhilm) [inline packages/integration/lib/src/widgets/lists/feed/feed_list.dart:153]: We can use the widget from widgetUtility right?

File: packages/integration/lib/src/views/post/widgets/delete_dialog.dart [hunk 11]
```diff
@@ -256,7 +262,7 @@ class LMFeedDeleteConfirmationDialog extends StatelessWidget {
                   onPressed: () {
                     Navigator.of(context).pop();
                   },
-                  child:  Text(
+                  child: Text(
```
Comment (divyanshgandhilm) [inline packages/integration/lib/src/views/post/widgets/delete_dialog.dart:265]: Use `LMText`

File: packages/integration/lib/src/views/views.dart [hunk 12]
```diff
@@ -2,6 +2,17 @@ export 'post/post_detail_screen.dart';
...
+export 'report/report_screen.dart';
 
 export 'activity/activity_screen.dart';
 export 'activity/user_activity_screen.dart';
+export 'activity/notification_screen.dart';
+
+export 'search/search_screen.dart';
+export 'media/media_preview_screen.dart';
+
+export 'post/widgets/delete_dialog.dart';
```
Comment (divyanshgandhilm) [inline packages/integration/lib/src/views/views.dart:17]: should be exported from widgets.dart

File: packages/integration/lib/src/views/views.dart [hunk 13]
```diff
@@ -2,6 +2,17 @@ export 'post/post_detail_screen.dart';
 export 'compose/compose_screen.dart';
 export 'likes/likes_screen.dart';
 export 'feed/feed_screen.dart';
+export 'feed/topic_select_screen.dart';
+
+export 'report/report_bottom_sheet.dart';
```
Comment (divyanshgandhilm) [inline packages/integration/lib/src/views/views.dart:7]: should be exported from widgets.dart

**Review by divyanshgandhilm — CHANGES_REQUESTED** [review]

File: packages/integration/lib/src/views/activity/notification_screen.dart (hunk 2 above)
Comment (TyagiLikeMinds) [reply packages/integration/lib/src/views/activity/notification_screen.dart:257]: No, this function is related to notification feed and not push notifications

File: packages/integration/lib/src/views/media/media_preview_screen.dart [hunk 14]
```diff
@@ -25,6 +25,9 @@ class LMFeedMediaPreviewScreen extends StatefulWidget {
 }
 
 class _LMFeedMediaPreviewScreenState extends State<LMFeedMediaPreviewScreen> {
+  final DateFormat formatter = DateFormat('MMMM d, hh:mm');
```
[the remark this answers is not in the document]
Comment (TyagiLikeMinds) [reply packages/integration/lib/src/views/media/media_preview_screen.dart:28]: Media Preview Screen currently do not have any customisability, We can revamp the whole screen in the coming releases?

File: packages/integration/lib/src/views/media/media_preview_screen.dart (hunk 3 above)
Comment (TyagiLikeMinds) [reply packages/integration/lib/src/views/media/media_preview_screen.dart:67]: This screen do not have any customisability as of now, we can align it in the coming release

File: packages/integration/lib/src/widgets/lists/feed/feed_list.dart (hunk 10 above)
Comment (TyagiLikeMinds) [reply packages/integration/lib/src/widgets/lists/feed/feed_list.dart:153]: We are using widgets from widget utility, some widgets need to access state variable that is why we have def widgets in the screen itself

**Review by divyanshgandhilm — DISMISSED** [review]

**Review by divyanshgandhilm — APPROVED** [review]

**Pull request merged**